python/setup.py

```diff
@@ -15,26 +18,38 @@
     from setuptools import setup
     from setuptools.extension import Extension
 
+
 class BinaryDistribution(Distribution):
     def has_ext_modules(self):
         return True
 
+
 CURRENT_DIR = os.path.dirname(__file__)
 
+
 def get_lib_path():
     """Get library path, name and version"""
-     # We can not import `libinfo.py` in setup.py directly since __init__.py
+    # We can not import `libinfo.py` in setup.py directly since __init__.py
     # Will be invoked which introduces dependences
     libinfo_py = os.path.join(CURRENT_DIR, './dgl/_ffi/libinfo.py')
     libinfo = {'__file__': libinfo_py}
-    exec(compile(open(libinfo_py, "rb").read(), libinfo_py, 'exec'), libinfo, libinfo)
+    exec(
+        compile(
+            open(
+                libinfo_py,
+                "rb").read(),
+            libinfo_py,
+            'exec'),
+        libinfo,
+        libinfo)
```

Comment:
I noticed that there is a lot of changes that breaks one like into multiple lines.  What was the PEP 8 message for this?  If the message is "line exceeding 80 characters", note that our limit is 100 characters (https://github.com/dmlc/dgl/blob/master/tests/lint/pylintrc).